posthog/temporal/workflows/clickhouse.py

```diff
@@ -36,17 +297,16 @@ async def get_client():
     #        ssl_context.load_verify_locations(settings.CLICKHOUSE_CA)
     #    elif ssl_context.verify_mode is ssl.CERT_REQUIRED:
     #        ssl_context.load_default_certs(ssl.Purpose.SERVER_AUTH)
-    timeout = ClientTimeout(total=None, connect=None, sock_connect=None, sock_read=None)
-    with TCPConnector(verify_ssl=False) as connector:
-        async with ClientSession(connector=connector, timeout=timeout) as session:
-            client = ChClient(
+    timeout = aiohttp.ClientTimeout(total=None, connect=None, sock_connect=None, sock_read=None)
+    with aiohttp.TCPConnector(ssl=False) as connector:
```

Comment:
Do the production clickhouse allow connecting without SSL?

Reply:
Yeah, it's what we have been doing so far.

Reply:
Don't ask me why:
1. It works without SSL.
2. It doesn't work with SSL CER_REQUIRED.